server/routes/place/html.py

```diff
@@ -73,20 +73,27 @@ def place(place_dcid=None):
     place_name = place_dcid
 
   place_summary = None
-  if os.environ.get('FLASK_ENV') in ['autopush', 'local']:
+  show_summary = False
+  if os.environ.get('FLASK_ENV') in ['autopush', 'local', 'staging', 'prod']:
     if not category:
       # Only show summary for Overview
       place_summary = current_app.config['PLACE_EXPLORER_SUMMARIES'].get(
```

Comment:
looks like PLACE_EXPLORER_SUMMARIES already have the allowed list as the key, why still need json for PLACE_SUMMARY_ALLOW_LIST?

Reply:
Was originally thinking we would have separate entries: one EXPLORER_SUMMARIES with content that would show on _autopush_ while we develop, and ALLOW_LIST are the subset of summaries allowed on _prod_. Would we prefer it to be just one file?

Reply:
I see, the if condition here is a bit confusing. From what you explain, I feel the condition should be:


show_summary = (place_dcid in place_allow_list) or (place_dcid in current_app.config['PLACE_EXPLORER_SUMMARIES']) # check prod then check other environment
place_summary = current_app.config['PLACE_EXPLORER_SUMMARIES'][place_dcid]

Reply:
Good call out. Reworked the if conditions to make the logic more clear.